client/layout/universal-navbar-footer/index.tsx

```diff
@@ -277,7 +277,7 @@ const UniversalNavbarFooter = () => {
 													{ translate( 'Get it on' ) }
 												</span>
 												<span className="lp-app-button__line lp-app-button__line--bottom">
-													{ translate( 'Google Play' ) }
+													Google Play
```

Comment:
"Google Play" should not be localized: https://play.google.com/intl/en_us/badges/